src/tests/ngraph_helpers/ngraph_functions/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ addIeTarget(
                 openvino::runtime
                 ngraph_reference
                 interpreter_backend
+                inference_engine_transformations
```

Comment:
double linking. Please, choose only one place where you want to link the lib (shared lib or ngraph functions). Ngrpah functions is linked by shared lib.

Reply:
fixed. please check again. thanks a lot!

Reply:
why did we change this?

Reply:
@rkazants The private ops augrucell and augrusquence are from "inference_engine_transformations". If without the target link, compiling will happen as "makeAUGRU" added in ngraph_functions.

Reply:
Cecilia, I see that we put this internal operation into common transformation part. Is it correct that we didn't place it into core part where we have public operations?

Reply:
@rkazants I remember the team talked of place for private ops before. Unfortunately they are not in the core part. @mbencer